code/modules/gear_presets/synths.dm

```diff
@@ -262,6 +262,8 @@
 		WEAR_FEET = /obj/item/clothing/shoes/galoshes
 	)
 
+	survivor_variant = CIVILIAN_SURVIVOR
```

Comment:
Please look at the main /datum/equipment_preset/synth/survivor
It's already a civilian variant, any subtype of that will be civilian unless stated otherwise, you are simply adding redundant bloat to these equipment preset subtypes when they were already considered civilian